convert2rhel/systeminfo.py

```diff
@@ -105,12 +108,14 @@ def resolve_system_info(self):
         self.id = self.name.split()[0].lower()
         self.version = self._get_system_version()
         self.arch = self._get_architecture()
+        self.has_internet_access = self._check_internet_access()
```

Comment:
Please move the function call down, e.g. under `self.booted_kernel = self._get_booted_kernel()` to not break the output this way:
```
[05/17/2022 17:28:38] TASK - [Prepare: Gather system information] *******************************
Name:                Oracle Linux Server
OS version:          8.4
Architecture:        x86_64
Checking internet connectivity using host '8.8.8.8' and port '53'.
Config filename:     oracle-8-x86_64.cfg
```

Reply:
done.